WordPress/src/main/java/org/wordpress/android/ui/uploads/UploadService.java

```diff
@@ -56,6 +56,15 @@
     @Inject PostStore mPostStore;
     @Inject SiteStore mSiteStore;
 
+    private class UploadingPost {
+        PostModel postModel;
+        List<MediaModel> pendingMedia = new ArrayList<>();
+        UploadingPost(PostModel postModel, List<MediaModel> pendingMedia) {
+            this.postModel = postModel;
+            this.pendingMedia = pendingMedia;
+        }
+    }
+
```

Comment:
As we discussed, keeping `UploadingPost` as we're now also tracking 'cancelled' status for posts with pending media (1c759a8) using it.